### PR TITLE
Fix for bug 1195652

### PR DESCRIPTION
**Pull request by anivan-suse**
#### Description:

Rule: aide_check_audit_tools

#### Rationale:
- Fixes # bug 1195652: https://bugzilla.suse.com/show_bug.cgi?id=1195652

### CODE REVIEW

File: linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh [hunk 1]
```diff
@@ -18,6 +18,12 @@
 {{% set auditfiles = auditfiles + ["/usr/sbin/audispd"] %}}
 {{% endif %}}
 
+{{% if 'rhel' not in product %}}
+{{% set configString = 'p+i+n+u+g+s+b+acl+selinux+xattrs+sha512' %}}
+{{% else %}}
+{{% set configString = "p+i+n+u+g+s+b+acl+xattrs+sha512" %}}
+{{% endif %}}
+
 {{% set configString = "p+i+n+u+g+s+b+acl+xattrs+sha512" %}}
```
Comment (yuumasato) [inline linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh:27]: I think you'll need to remove definition of `configString` in line 27.

File: linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh [hunk 2]
```diff
@@ -18,6 +18,12 @@
 {{% set auditfiles = auditfiles + ["/usr/sbin/audispd"] %}}
 {{% endif %}}
 
+{{% if 'rhel' not in product %}}
```
Comment (yuumasato) [inline linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh:21]: I'm not sure how valid or expected this change is to all products.
Is your change in the context of SLE only? Or all products that are not RHEL?

If in context of SLE, suggest the following, as this keeps the `configString` the same for the rest of products.
```suggestion
{{% if 'sle' in product %}}
```

**Review by yuumasato — CHANGES_REQUESTED** [review]

File: linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh (hunk 1 above)
Comment (anivan-suse) [reply linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh]: Done!

**Review by yuumasato — APPROVED** [review]
Thank you for the fix, the remediation is now aligned with the Ansible one.

**Pull request merged**